api/monitoring_instance.go

```diff
@@ -1,352 +0,0 @@
-			skipVerifyTLS,
-		)
-		if err != nil {
-			e.l.Error(err)
-			return ctx.JSON(http.StatusInternalServerError, Error{
-				Message: pointer.ToString("Could not create an API key in PMM"),
-			})
-		}
-	}
-	if apiKey != "" {
-		_, err = e.kubeClient.UpdateSecret(c, &corev1.Secret{
-			ObjectMeta: metav1.ObjectMeta{
-				Name:      name,
-				Namespace: namespace,
-			},
-			Type:       corev1.SecretTypeOpaque,
-			StringData: e.monitoringConfigSecretData(apiKey),
-		})
-		if err != nil {
-			e.l.Error(err)
-			return ctx.JSON(http.StatusInternalServerError, Error{
-				Message: pointer.ToString(fmt.Sprintf("Could not update k8s secret %s", name)),
-			})
-		}
-	}
-	if params.Url != "" {
-		m.Spec.PMM.URL = params.Url
-	}
-	if params.AllowedNamespaces != nil {
-		m.Spec.AllowedNamespaces = *params.AllowedNamespaces
-	}
-	if params.VerifyTLS != nil {
-		m.Spec.VerifyTLS = params.VerifyTLS
-	}
-	err = e.kubeClient.UpdateMonitoringConfig(c, m)
-	if err != nil {
-		e.l.Error(err)
-		return ctx.JSON(http.StatusInternalServerError, Error{
-			Message: pointer.ToString("Failed updating monitoring instance"),
-		})
-	}
-
-	return ctx.JSON(http.StatusOK, &MonitoringInstance{
-		Type:              MonitoringInstanceBaseWithNameType(m.Spec.Type),
-		Name:              m.GetName(),
-		Namespace:         m.GetNamespace(),
-		Url:               m.Spec.PMM.URL,
-		AllowedNamespaces: &m.Spec.AllowedNamespaces,
-		VerifyTLS:         m.Spec.VerifyTLS,
-	})
-}
-
-// DeleteMonitoringInstance deletes a monitoring instance.
-func (e *EverestServer) DeleteMonitoringInstance(ctx echo.Context, namespace, name string) error {
-	used, err := e.kubeClient.IsMonitoringConfigUsed(ctx.Request().Context(), namespace, name)
-	if err != nil {
-		if k8serrors.IsNotFound(err) {
-			return ctx.JSON(http.StatusNotFound, Error{
-				Message: pointer.ToString("Monitoring instance is not found"),
-			})
-		}
-		e.l.Error(err)
-		return ctx.JSON(http.StatusInternalServerError, Error{
-			Message: pointer.ToString("Failed to check the monitoring instance is used"),
-		})
-	}
-	if used {
-		return ctx.JSON(http.StatusBadRequest, Error{
-			Message: pointer.ToString(fmt.Sprintf("Monitoring instance %s is used", name)),
-		})
-	}
-	if err := e.kubeClient.DeleteMonitoringConfig(ctx.Request().Context(), namespace, name); err != nil {
-		if k8serrors.IsNotFound(err) {
-			return ctx.JSON(http.StatusNotFound, Error{
-				Message: pointer.ToString("Monitoring instance is not found"),
-			})
-		}
-		e.l.Error(err)
-		return ctx.JSON(http.StatusInternalServerError, Error{
-			Message: pointer.ToString("Failed to get monitoring instance"),
-		})
-	}
-	if err := e.kubeClient.DeleteSecret(ctx.Request().Context(), namespace, name); err != nil {
-		if k8serrors.IsNotFound(err) {
-			return ctx.NoContent(http.StatusNoContent)
-		}
-		e.l.Error(err)
-		return ctx.JSON(http.StatusInternalServerError, Error{
-			Message: pointer.ToString("Failed deleting monitoring instance"),
-		})
-	}
```

Comment:
Where did this code go? Is this now handled by the everest-operator?

Reply:
Its not, I just happened to have missed moving this code. Thanks for catching it!